build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +53,7 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable flexible IPAM mode for Antrea. This mode allows to assign subnets to deployments and
+# stateful sets via IP Pool annotation.
+# flexibleIPAM: false
```

Comment:
We also need to add feature gate to antrea_features.go
The option compatibility verification can be added in future.

Reply:
adding

Reply:
Done